### ISSUE TITLE
Entrega 3 Apartado 3.2 : Duda enunciado 3.2 segundo punto

### ISSUE THREAD

**Issue by mariizamb** [IIC2413/Syllabus-2021-2#378]
Hola ! Una duda corta, en el segundo punto del 3.2 cuando dicen "separar claramente e indicar puntuación y cantidad de visualizaciones de cada una", con puntuación se refieren a quien quedó en primero, segundo y tercer lugar? o a que se refiere?
Gracias!
Saludos

**Comment by arpincheira** [comment]
No, se refiere al **atributo puntuación** de las películas y series. Ahora, con separar se refiere a separar las películas de las series e indicar claramente que output que corresponde a series y cual a películas.
